apps/web/pages/v2/apps/[slug]/setup.tsx

```diff
@@ -0,0 +1,41 @@
+import { GetStaticPaths, InferGetStaticPropsType } from "next";
+import { useSession } from "next-auth/react";
+import { useRouter } from "next/router";
+
+import { getStaticProps } from "@calcom/app-store/_pages/setup/_getStaticProps";
+import { AppSetupPage } from "@calcom/app-store/_pages/v2/setup";
+import Loader from "@calcom/ui/Loader";
+
+export default function SetupInformation(props: InferGetStaticPropsType<typeof getStaticProps>) {
+  const router = useRouter();
+  const slug = router.query.slug as string;
+  const { status } = useSession();
+
+  if (status === "loading") {
+    return (
+      <div className="absolute z-50 flex h-screen w-full items-center bg-gray-200">
+        <Loader />
+      </div>
+    );
```

Comment:
I believe we wanna use skeletons in the future for everything, no? @zomars

Reply:
Agreed @PeerRich but in this context I don't see the benefit of using skeletons

It's just a blank setup page

<img width="1152" alt="image" src="https://user-images.githubusercontent.com/3504472/188227453-71cfd7e3-585d-464e-a4ec-81475d3baf78.png">